Files/DataStandard/Vault.Custom/addinVault/powerGateMain.ps1

```diff
@@ -222,20 +222,26 @@ function RefreshView {
 
 function SetEntityProperties($erpMaterial, $vaultEntity) {
 	#TODO: Update Entity UDPs with values from ERP
-	if ($vaultEntity._EntityTypeID -eq "ITEM") { 
-		$vaultEntity = Update-VaultItem -Number $vaultEntity._Number -NewNumber $erpMaterial.Number
-		Update-VaultItem -Number $vaultEntity._Number -Properties @{
-			#the item description cannot be updated, since "Description (Item,CO)" is a system property!
-			"_Description(Item,CO)" = $erpMaterial.Description
+	if ($vaultEntity._EntityTypeID -eq "ITEM") {
+		try {
+			Update-VaultItemWithErrorHandling -Number $vaultEntity._Number -Properties @{
```

Comment:
Is this code RENAMING the vault item?
Because in the removed code `-NewNumber $erpMaterial.Number` was used, can you test it if the Vault Item is renamed when a erp item is created for an vault item. @TollJulian